README.txt

```diff
@@ -71,6 +71,12 @@ The Payment Banking Slip Pix for LifterLMS is now activated.
 
 == Changelog ==
```

Comment:
Não é desnecessária a área de changelog dentro do readme como já temos o `CHANGELOG.md`?
Não é possível também utilizar apenas o `README.md` para ambos o git e o [lançamento no Wordpress](https://johnblackbourn.com/readme-md-plugin-on-wordpress-org/)?

Reply:
Infelizmente não é possível fazer essa alteração segundo os [guidelines do WordPress](https://developer.wordpress.org/plugins/wordpress-org/plugin-developer-faq/#are-there-specific-things-that-i-should-avoid-doing), eles exigem que seja um arquivo README.txt para que seja listado corretamente na plataforma oficial